rfc8826.xml

```diff
@@ -1214,70 +1380,133 @@ the title) for use on https://www.rfc-editor.org/search. -->
+<https://www.w3.org/TR/cors/>:
+
+[CORS]
+    Anne van Kesteren. Cross-Origin Resource Sharing. 2 June 2020. REC. URL:
+    https://www.w3.org/TR/cors/ 
+
+<https://www.w3.org/TR/2020/SPSD-cors-20200602/> says that new implementations
+should follow the "Fetch API Living Standard".
+
+Please review and let us know if any updates are needed. 
+-->
+
+        <reference anchor="CORS">
           <front>
             <title>Cross-Origin Resource Sharing</title>
             <author initials="A." surname="van Kesteren">
               <organization/>
             </author>
-            <!-- Date from http://www.w3.org/TR/2014/REC-cors-20140116/ -->
-            <date day="16" month="January" year="2014"/>
+            <date month="January" year="2014"/>
           </front>
         </reference>
 
         <reference anchor="SWF" target="http://www.adobe.com/content/dam/Adobe/en/devnet/swf/pdf/swf_file_format_spec_v10.pdf">
           <front>
             <title>SWF File Format Specification Version 19</title>
-            <author surname="Adobe">
-              <organization/>
-            </author>
-            <!-- Date from PDF properties -->
-            <date day="23" month="April" year="2013"/>
+            <author/>
+            <date month="April" year="2013"/>
           </front>
         </reference>
 
-        <reference anchor="XmlHttpRequest" target="http://www.w3.org/TR/XMLHttpRequest/">
+<!-- [rfced] Informative References:  
+The URL provided for [SWF] in the original XML file -
+<http://www.adobe.com/content/dam/Adobe/en/devnet/swf/pdf/
+swf_file_format_spec_v10.pdf> - steers to
+<https://www.adobe.com/content/dam/acom/en/devnet/swf/pdf/
+swf_file_format_spec_v10.pdf>, which in turn yields a 404.
+Please provide a working and stable URL.
+
+Original:
+ [SWF]      "SWF File Format Specification Version 19", April 2013. -->
+
+        <reference anchor="XmlHttpRequest" target="https://www.w3.org/TR/XMLHttpRequest/">
           <front>
             <title>XMLHttpRequest Level 2</title>
             <author initials="A." surname="van Kesteren">
               <organization/>
             </author>
-            <date day="17" month="January" year="2012"/>
+            <date month="January" year="2012"/>
           </front>
         </reference>
 
+<!-- [rfced] Informative References:  The URL as provided for
+[XmlHttpRequest] in the original document -
+<http://www.w3.org/TR/XMLHttpRequest/> - steers to a page with the
+title "XMLHttpRequest Level 1," dated October 2016.  When we did a
+Google search for "XMLHttpRequest Level 2," we found
+<https://www.w3.org/TR/2012/WD-XMLHttpRequest-20120117/>, which is
+partially obscured by a red box that says "This version is
+outdated!"  The link in the box in turn steers to the October 2016
+"XMLHttpRequest Level 1" page.
+
+Please advise.
+
+Original:
+ [XmlHttpRequest]
+            van Kesteren, A., "XMLHttpRequest Level 2", January 2012. -->
```

Comment:
Ask Anne

Reply:
@annevk: can you provide a proposed Fetch reference.

Reply:
If you want to continue talking about XMLHttpRequest, this could be:

```[XMLHTTPREQUEST]

    Anne van Kesteren. XMLHttpRequest. Living Standard. URL:
    https://xhr.spec.whatwg.org/
```
But you could also replace the XMLHttpRequest examples with `fetch()` examples and use the Fetch reference I suggested above.